install_workflow/create_config_file.py

```diff
@@ -19,7 +19,7 @@
   "bin_process_path": bin_process_path,
   "qcore_lib_path": qcore_path,
   "install_bb_name": "install_bb.py",
-  "params_vel": "params_vel.yaml",
+  "vm_params": "vm_params.yaml",
```

Comment:
Since we are adding dependencies towards qcore.constant,  i think its a good time to remove this parameter from this config.